### PR TITLE
Add html form

### PR DESCRIPTION
**Pull request by SuprunArtem**

### CODE REVIEW

**Comment by SuprunArtem** [comment]
Add html form

File: src/index.html [hunk 1]
```diff
@@ -2,16 +2,184 @@
...
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      
+      <fieldset class="fieldset">
```
Comment (DarkMistyRoom) [inline src/index.html]:
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
      <fieldset class="fieldset">
```

Comment (DarkMistyRoom) [reply src/index.html]: Don't add an empty line between parent and child

File: src/index.html [hunk 2]
```diff
@@ -2,16 +2,184 @@
...
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input 
+          type="text"
```
Comment (DarkMistyRoom) [inline src/index.html]:
```suggestion
          <input 
            type="text"
```

File: src/index.html [hunk 3]
```diff
@@ -2,16 +2,184 @@
...
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        </label>
+
+        <label class="form-field">
+
+          <label>
```
Comment (DarkMistyRoom) [inline src/index.html]: Don't use this common label, because when you click on the question, yes becomes selected

File: src/index.html [hunk 4]
```diff
@@ -2,16 +2,184 @@
...
+             type="time"
+             name="goToBed"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select name="brandCar" multiple>
+            <option value="Bmw">Bmw
+            </option>
```
Comment (DarkMistyRoom) [inline src/index.html]:
```suggestion
            <option value="Bmw">
              Bmw
            </option>
```

File: src/index.html [hunk 5]
```diff
@@ -2,16 +2,184 @@
...
+            <option value="Audi">Audi
+            </option>
+            <option value="Lada">Lada
+            </option>
+          </select>
+        </label>
+
+        <label class="form-field">
+            How do you rate our work?
+          <input
```
Comment (DarkMistyRoom) [inline src/index.html]:
```suggestion
        <label class="form-field">
          How do you rate our work?
          <input
```

Comment (DarkMistyRoom) [reply src/index.html]: fix it everywhere

File: src/index.html [hunk 6]
```diff
@@ -2,16 +2,184 @@
...
+        </label>
+      </fieldset> 
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          <label>
+                 Comments:
+            <textarea name="Comments">
+            </textarea>
```
Comment (DarkMistyRoom) [inline src/index.html]: image

File: src/index.html [hunk 7]
```diff
@@ -2,16 +2,184 @@
...
+          </label>
+        </label>
+      </fieldset>
+      <button name="button" type="submit">Submit
+      </button>
+    </form>
     <script type="text/javascript" src="./main.js"></script>
   </body>
-</html>
+</html>
```
Comment (DarkMistyRoom) [inline src/index.html]: add the empty line

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Add links to description. and push the readme file

File: src/index.html [hunk 8]
```diff
@@ -2,16 +2,188 @@
...
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+        An interesting fact about you!
+        </legend>
+        </label>
+
+        <label class="form-field">
+
```
Comment (SanyaBratashchuk) [inline src/index.html]:
```suggestion
```

File: src/index.html [hunk 9]
```diff
@@ -2,16 +2,188 @@
...
+              name="Comments"
+              id="comments"
+           ></textarea>
+          </label> 
+        </label>
+    
+        <label class="form-field">
+          <label>
+            Would you recommended us?
+            <select    name="recommended">
```
Comment (SanyaBratashchuk) [inline src/index.html:175]:
```suggestion
            <select name="recommended">
```

File: src/index.html [hunk 10]
```diff
@@ -2,16 +2,188 @@
...
+        </label>
+      </fieldset>
+      <button name="button" type="submit">Submit
+      </button>
+    </form>
     <script type="text/javascript" src="./main.js"></script>
   </body>
-</html>
+
+</html>
```
Comment (SanyaBratashchuk) [inline src/index.html]:
```suggestion
</html>

```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Review by anstsot — CHANGES_REQUESTED** [review]
Please check the demo link and report links. they are not working
and please fix all comments above

**Review by polosanya — CHANGES_REQUESTED** [review]
Your links still don't work properly. Feel free to ask for help in the chat if you have any difficulties

**Pull request closed without merging**